static/src/javascripts/projects/commercial/modules/header-bidding/prebid/prebid.ts

```diff
@@ -330,7 +331,14 @@ const initialise = (window: Window, framework = 'tcfv2'): void => {
 		 * */
 		advert.hasPrebidSize = true;
 
-		if (data.bidderCode === 'improvedigital') captureCommercialMetrics();
+		/**
+		 * If improve skin has won auction, toggle force send metrics on.
+		 * TODO remove after improve skin release.
+		 * */
+		if (data.bidderCode === 'improvedigital') {
+			setForceSendMetrics(true);
+			captureCommercialMetrics();
+		}
```

Comment:
[Looking at the Prebid code][1], it seems like we could actually extract this into its own `window.pbjs.onEvent` function, which takes and `id` as the third parameter.

Could be a little cleaner, maybe?

```ts
window.pbjs.onEvent(
	'bidWon',
	() => {
		setForceSendMetrics(true);
		captureCommercialMetrics();
	},
	'improvedigital', // this ensures it only runs for this bidder
);
``` 

[1]: https://github.com/prebid/Prebid.js/blob/ebaa5e55cc4e4b4737167493459849cb2615eac6/src/prebid.js#L681